### PR TITLE
Added FAQ section 

### PR DESCRIPTION
**Pull request by Croooook**
- [x] added connectors section in FAQ.
- [x] added resolver section in FAQ.
- [x] added dsa-connect section in FAQ.
- [x] added miscellaneous section in FAQ.
- [x] added terminal section in FAQ.
- [ ] added some questions respective to every section.

### CODE REVIEW

**Comment by KABBOUCHI** [comment]
please can u remove `FAQ` and `-` from menu titles?

image

File: content/en/FAQ/connectors/getid-and-setid.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+`setId` and `getId` will help you share data/value from one spell to another. You can get a value from a specific `Id` that had been stored before.
+
+**Example where getId and setId can be used**
+
+> Let's say I want to sell USDC to buy ETH on 1Inch and then deposit bought ETH into Compound.
+1) So I sell USDC on 1Inch, and I store the bought ETH amount at `setId` (let's say it's `482` as setId for now) on `sell()` function of 1inch. 
+2) Then I will use `482` as getId on compound deposit() function and deposit the bought ETH in Compound. In deposit function, I don't need to mention the amount specifically to deposit. It will get the deposit amount from `getId` that is passed. 
+
+**Code Snippet:**
+```
```
Comment (KABBOUCHI) [inline content/en/FAQ/connectors/getid-and-setid.md:19]: use `js` for code highlights 
````
```js
    ....

```
````

**Review by KABBOUCHI — CHANGES_REQUESTED** [review]

File: content/en/faq/terminal/typechain-guide.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+   const gas = await web3.eth.getGasPrice();
+}
+
+run()
+ .then(() => console.log("Done"))
+ .catch((error) => console.log(error.message))
+```
+But we have top level await, no need to any wrapper
+```js
+const gas = await
```
Comment (KABBOUCHI) [inline content/en/faq/terminal/typechain-guide.md:39]: plz can u put them in 1 line?

**Review by KABBOUCHI — CHANGES_REQUESTED** [review]

File: content/en/faq/terminal/typechain-guide.md (hunk 2 above)
Comment (Croooook) [reply content/en/faq/terminal/typechain-guide.md:39]: put them what.. I didn't get this.. the white statement is a normal explanation .. of things and below is the code!

**Comment by Sowmayjain** [comment]
@KABBOUCHI let's merge this?

File: content/en/faq/terminal/typechain-guide.md (hunk 2 above)
Comment (KABBOUCHI) [reply content/en/faq/terminal/typechain-guide.md:39]:
```js
const gas = await 
web3.eth.getGasPrice();
```
=> 
```js
const gas = await web3.eth.getGasPrice();
```

File: content/en/faq/terminal/typechain-guide.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+**Libraries**
+> 1) BigNumber
+> 2) axios
+> 3) DSA
+> 4) Web3
+> 5) wait
+
+**Constants**
+> 1) MAINNET_TOKENS
```
Comment (KABBOUCHI) [inline content/en/faq/terminal/typechain-guide.md:16]: plz can u add `AVALANCHE_TOKENS` & `ARBITRUM_TOKENS`

**Review by KABBOUCHI — CHANGES_REQUESTED** [review]

File: content/en/faq/terminal/typechain-guide.md (hunk 2 above)
Comment (Croooook) [reply content/en/faq/terminal/typechain-guide.md:39]: done

**Comment by Croooook** [comment]
@KABBOUCHI  @Sowmayjain  I have done all the changes, Now I think it's ready to be merged. Pls review

**Comment by KABBOUCHI** [comment]
> @KABBOUCHI @Sowmayjain I have done all the changes, Now I think it's ready to be merged. Pls review

plz can u remove empty faq pages?
image

**Comment by Croooook** [comment]
> > @KABBOUCHI @Sowmayjain I have done all the changes, Now I think it's ready to be merged. Pls review
> 
> plz can u remove empty faq pages? image

done!

**Review by KABBOUCHI — APPROVED** [review]

**Pull request merged**